From 282ee759beef7b3656a3504fbcae0b3242501b03 Mon Sep 17 00:00:00 2001
From: xIsm4 <minelatinsoporte@gmail.com>
Date: Tue, 1 Mar 2022 01:06:26 +0100
Subject: [PATCH] Remove server suspend


diff --git a/src/main/java/org/bukkit/craftbukkit/CraftServer.java b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
index 52b2d106..c7113dc9 100644
--- a/src/main/java/org/bukkit/craftbukkit/CraftServer.java
+++ b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
@@ -564,6 +564,7 @@ public final class CraftServer implements Server {
         return BukkitConfig.ticksPerMonsterSpawn;
     }
 
+    /*
     @Override
     public @Nullable Duration getEmptyServerSuspendDelay() {
         try {
@@ -573,6 +574,7 @@ public final class CraftServer implements Server {
             return null;
         }
     }
+     */
 
     @Override
     public PluginManager getPluginManager() {
diff --git a/src/main/java/org/github/paperspigot/PaperSpigotConfig.java b/src/main/java/org/github/paperspigot/PaperSpigotConfig.java
index 8d58d1f3..38a3bbea 100644
--- a/src/main/java/org/github/paperspigot/PaperSpigotConfig.java
+++ b/src/main/java/org/github/paperspigot/PaperSpigotConfig.java
@@ -128,10 +128,12 @@ public class PaperSpigotConfig
         tickEmptyWorlds = getBoolean("settings.tick-empty-worlds", tickEmptyWorlds);
     }
 
+    /*
     public static String emptyServerSuspendDelay = "PT1M";
     private static void setEmptyServerSuspendDelay() {
         emptyServerSuspendDelay = getString("settings.empty-server-suspend", emptyServerSuspendDelay);
     }
+     */
 
     public static boolean requireAllPlugins = false;
     private static void requireAllPlugins() {
-- 
2.34.1.windows.1

